bazel/envoy_library.bzl

```diff
@@ -80,6 +80,8 @@ EXTENSION_STATUS_VALUES = [
 def envoy_cc_extension(
         name,
         security_posture,
+        # Make this mandatory once all extensions have had their cat added.
+        category = None,
```

Comment:
@htuch so iiuc we need this (altho made mandatory)

and...

Reply:
Yep